devices/__init__.py

```diff
@@ -19,7 +19,141 @@
 
 from __future__ import absolute_import
 import importlib
-from core import CanProcess
+from core import CanProcess, CanProcessComposite, StateMachine
+from core.utils import dict_strict_update
+
+
+class Device(CanProcess):
```

Comment:
From the point of view of the device authors, this is going to make far more sense. Even though it's just a name change.